src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -296,7 +296,7 @@ namespace CommandLineInterface {
         state.dataStrGlobals->inputDirPath = FileSystem::getParentDirectoryPath(state.dataStrGlobals->inputFilePath);
 
         {
-            auto const fileType = FileSystem::getFileType(state.dataStrGlobals->inputFilePath);
+            auto const &fileType = FileSystem::getFileType(state.dataStrGlobals->inputFilePath);
```

Comment:
This is a scalar so no `auto` and no `&`. Use `FileSystem::FileTypes`. Why is the `enum` called `FileTypes` and not `FileType`?